data/core/editor/items.cfg

```diff
@@ -186,24 +265,54 @@
         name= _ "Open Plain Chest"
     [/item]
 
-    [item]
-        id=chest
-        image=items/chest.png
-        name= _ "Chest"
-    [/item]
-
     [item]
         id=cloak-green
         image=items/cloak-green.png
         name= _ "Green Cloak"
     [/item]
 
+    [item]
+        id=club
+        image=items/club.png
+        name= _ "Club"
+    [/item]
+
     [item]
         id=coffin-closed
         image=items/coffin-closed.png
         name= _ "Closed Coffin"
     [/item]
 
+    [item]
+        id=coffin-open
+        image=items/coffin-open.png
+        name= _ "Open Coffin"
+    [/item]
+
+    [item]
+        id=crossbow
+        image=items/crossbow.png
+        name= _ "Crossbow"
+    [/item]
+
+    [item]
+        id=dagger
+        image=items/dagger.png
+        name= _ "Dagger"
+    [/item]
+
+    [item]
+        id=dagger-poison
+        image=items/dagger-poison.png
+        name= _ "Poison Dagger"
+    [/item]
+
+    [item]
+        id=dagger-poison-bare
+        image=items/dagger-poison-bare.png
+        name= _ "Bare Poison Dagger"
```

Comment:
It's more "a normal dagger, which happens to match the image used for the Poison Dagger, so would look in-place in an assassin's workshop where the poison is being added". Maybe "Poison Dagger (clean)"?

Po hint also needs to say that "bare" doesn't mean the verb "to pull out a knife and threaten someone with it".